src/display/annotation_layer.js

```diff
@@ -974,17 +974,23 @@ class WidgetAnnotationElement extends AnnotationElement {
     // If the height is "big" then it could lead to a too big font size
     // so in this case use the one we've in the pdf (hence the min).
     let computedFontSize;
+    const BORDER_SIZE = 2;
+    const fixDec = (x, n) => Math.round(10 ** n * x) / 10 ** n;
```

Comment:
Do you foresee additional usage of this helper, since otherwise I think that we can simplify it slightly (and perhaps improve the name)?
```suggestion
    const roundToOneDecimal = x => Math.round(10 * x) / 10;
```

---

This suggestion is based on a *similar* method found in https://github.com/mozilla/pdf.js/blob/782d098baf4eb556d7f8a62a10174aa34253dcca/src/core/annotation.js#L2097